examples/step-63/step-63.cc

```diff
@@ -151,8 +157,11 @@ namespace Step63
     bool                   output;
   };
 
+
+
   void Settings::get_parameters(const std::string &prm_filename)
   {
+    /* First declare the parameters... */
```

Comment:
```suggestion
    // First declare the parameters...
```

Reply:
No, I meant it that way. The issue is that tutorial programs become hard to read if the code is interrupted too many times by explanations. That's particularly the case if the comments are in nested if/else/for blocks because one visually loses the overview of the algorithm. So here I wanted to have these comments show up as part of the code block, rather than the running commentary. I think that makes it easier to understand the flow in this particular case.

Same for the next comment, of course.